run-cassandra-centos.sh

```diff
@@ -0,0 +1,366 @@
+#!/usr/bin/env bash
+
+if [[ -n "$TRACE" ]]; then
+    export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+    set -o xtrace
+fi
+set -euo pipefail
+
+IFS=$'\n\t'
+
+#===============================================================================
+# YCSB Parameters:
+#
+# Modify the following parameters based on how much you want to stress the Cassandra servers.
+#
+# RECORDCOUNT = The number of test records to be inserted into the Cassandra database at load time.
+# OPERATIONCOUNT = The number of test records to be manipulated (i.e. read/insert/update/delete) at run time.
+#    Set RECORDCOUNT and/or OPERATIONCOUNT to a higher number (as high as 10 millions) if you want to
+#    perform more stress testing. RECORDCOUNT does not have to match OPERATIONCOUNT.
+# THREADS = The number of test client threads to be run in parallel. Note: The maximum number of threads
+#    may depend on the CPU and Memory of your YCSB container.
+
+RECORDCOUNT=10000
+OPERATIONCOUNT=10000
+THREADS=10
+
+#===============================================================================
+
+IMAGE=
+PROFILE=
+PACKAGE=
+IMAGENAME=
+VERSION=
+UUID=
+NAME=
+IP1=
+IP2=
+TAG="test-instance=true"
+SCRIPT=$PWD/userscript.sh
```

Comment:
I think this and all the relevant code for METADATAFILE can be removed. It was used for testing that user-script and user-data works but I'm assuming the cassandra tests don't rely on either of those.